### PR TITLE
Before you start flow

### PR DESCRIPTION
**Pull request by davidrapson**
This PR:

- Tidies up the awards for all before you start page (still behind a flag), moving copy to locale files
- Moves start page before auth (this will change for the funding proposal form too)

### CODE REVIEW

File: assets/sass/utilities/_utilities-text.scss [hunk 1]
```diff
@@ -22,10 +22,7 @@
     h3,
     h4,
     h5,
-    h6 {
```
Comment (davidrapson) [inline assets/sass/utilities/_utilities-text.scss]: This is a global change but this style was a little unexpected and mean that `u-constrained-content` and `u-constrained-content-wide` now behave the same (with different max-widths)

**Comment by davidrapson** [comment]
[Add 'Before you start' page to NLA4A form](https://trello.com/c/7WRA0wkx/1018-add-before-you-start-page-to-nla4a-form)

File: assets/sass/utilities/_utilities-text.scss (hunk 1 above)
Comment (mattandrews) [reply assets/sass/utilities/_utilities-text.scss]: Makes sense – I saw this the other day and was a bit confused by it:

image

File: config/locales/en.yml [hunk 2]
```diff
@@ -1702,6 +1702,92 @@ applyNext:
...
+            <li>address the statements are sent to</li>
+            <li>your bank’s name</li>
+            <li>the account code and sort number</li>
+            <li>date the statement was issued.</li>
+        </ul>
+        <p>Here’s <a href="/assets/images/apply/afa-bank-statement-example.png">a handy picture of the kind of bank statement we’re looking for.</a></p>
+        <h3>5. We ask for information about what sort of project you’d like to do</h3>
+        <p>And how your project will help and involve your community.</p>
+        <h3>6. We also ask you to read and agree to our terms and conditions</h3>
+        <p>You can have a <a href="#">look at the terms and conditions here</a>.</p>
```
Comment (mattandrews) [inline config/locales/en.yml]: Where should this link go?

Comment (davidrapson) [reply config/locales/en.yml]: Good spot, have added that link now.

**Comment by davidrapson** [comment]
@mattandrews Mind giving this last commit a look? Added a feature flag so I can merge this without making the pre-login change live.

**Pull request merged**